decentraland/unity-explorer
# feat: Loading screen state

## What does this PR change?

Shows the loading screen status in the debug panel. It will show `current stage`, and if stage is scene loading, `assets loaded `and `asset to load`

![image](https://github.com/user-attachments/assets/72d7f107-545c-4a0c-aa61-efbccd3877b8)


## How to test the changes?

1. Launch the explorer
2. Check the debug panel to see the loading status

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
